### PR TITLE
Fix Rendered Sound. Authored by: @Manongjohn

### PR DESCRIPTION
**Pull request by JPUser1**
Fix Rendered Sound.

### CODE REVIEW

**Comment by manongjohn** [comment]
Still not doing the co-authored by correctly, but at least the PR doesn't have extra stuff in it anymore.

**Comment by JPUser1** [comment]
> 
so sorry I will get I will try to get it right still new with Github.

**Comment by RodneyBaker** [comment]
Testing this port now.  

REF:  https://github.com/tahoma2d/tahoma2d/pull/981

**Comment by konero** [comment]
Some pointers.

- When writing a pull request, include a description of what the problem was and why this solves it.
- In GitHub Desktop, you can add co-authors this way in the commit message box.


You can delete the unnecessary whitespace character on line 84 and commit to this branch including manongjohn this time, I think the commit messages will get merged together (?).

File: toonz/sources/toonzlib/movierenderer.cpp [hunk 1]
```diff
@@ -81,6 +81,10 @@ void getRange(ToonzScene *scene, bool isPreview, int &from, int &to) {
       int r0, r1;
       xs->getCellRange(k, r0, r1);
 
+       TXshColumn *col         = xs->getColumn(k);
```
Comment (konero) [inline toonz/sources/toonzlib/movierenderer.cpp:84]:
```suggestion
      TXshColumn *col         = xs->getColumn(k);
```

**Comment by manongjohn** [comment]
@RodneyBaker @shun-iwasawa 

Please let this serve as a formal request that if any of my changes in T2D are ported to OT and I am not properly credited in the `commit` message, then I ask that you do not merge it until the submitter fixes it **correctly**.

This ensures my contributions are properly recorded in the commit history. Crediting in the PR title/body is not the same thing as the repo can be moved and disassociated from the PRs but the commit history will always contain the original authors.

I ensure anything I port from OT to T2D is properly credited.  I expect the same consideration.

**Review by shun-iwasawa — CHANGES_REQUESTED** [review]
@JPUser1 Please try the following:

- Move to the working branch.

`> git checkout OpenToonz-Fix-rendered-sound-starting-too-soon`

- Start modifying the last commit message.

`> git commit --amend`

- Change the commit message, adding "Co-authored-by" like as follows:
```
Update movierenderer.cpp

Co-authored-by: manongjohn <dancel@ptd.net>
```
- Push the working branch again (with force push option).

`> git push -f origin OpenToonz-Fix-rendered-sound-starting-too-soon`

**Comment by manongjohn** [comment]
Please use: `manongjohn <19245851+manongjohn@users.noreply.github.com>`

**Comment by shun-iwasawa** [comment]
@manongjohn Modified the post. Thanks!

**Comment by JPUser1** [comment]
ok.

**Pull request closed without merging**